src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -164,6 +164,7 @@ class MulticopterPositionControl : public control::SuperBlock
 	control::BlockParamFloat _manual_thr_min;
 	control::BlockParamFloat _manual_thr_max;
 	control::BlockParamFloat _manual_land_alt;
+	control::BlockParamFloat _z_vel_man_expo;
```

Comment:
I started using BlockParam as it gives more readable results and I heard it's desired...